Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs

```diff
@@ -7,18 +7,34 @@
 using Promact.Trappist.DomainModel.Models.Category;
 using System;
 using Promact.Trappist.DomainModel.Models.Test;
-using System.Threading;
+using Microsoft.AspNetCore.Hosting;
+using Promact.Trappist.DomainModel.ApplicationClasses.BasicSetup;
+using Promact.Trappist.Utility.Constants;
 using System.Threading.Tasks;
+using System.Threading;
 
 namespace Promact.Trappist.DomainModel.DbContext
 {
     public class TrappistDbContext : IdentityDbContext<ApplicationUser>
     {
-        public TrappistDbContext(DbContextOptions<TrappistDbContext> options)
-            : base(options)
+        #region Private variables
+        #region Dependencies
+        private readonly IHostingEnvironment _environment;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private member. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1144)